c2rust-transpile/src/translator/mod.rs

```diff
@@ -4516,9 +4516,22 @@ impl<'c> Translation<'c> {
 
                 let is_const = pointee.qualifiers.is_const;
 
+                // Handle literals by looking at the next level of expr nesting. Avoid doing this
+                // for expressions that will be translated as const macros, because emitting the
+                // name of the const macro only occurs if we process the expr_id with a direct call
+                // to `convert_expr`.
```

Comment:
When/where does this happen?  That might've been part of what was tripping me up in adding `override_ty`s to const macros.

Reply:
Oh, sorry, I meant the opposite.  When do we not go through `fn convert_expr` to process an `expr_id`?